lighthouse-extension/app/src/extension-entry.js

```diff
@@ -180,23 +207,23 @@ if (typeof window !== 'undefined' && 'chrome' in window && chrome.runtime) {
 }
 
 if (typeof module !== 'undefined' && module.exports) {
-  // Export for importing types into popup.js, require()ing into unit tests.
+  // Export for importing types into popup.js and require()ing into unit tests.
```

Comment:
`require()ing into unit tests` is also a lie, but could be true someday :) Meanwhile this block is needed for the mentioned `popup.js` use